Tests/MoyaProviderIntegrationTests.swift

```diff
@@ -250,8 +251,7 @@ final class MoyaProviderIntegrationTests: QuickSpec {
                             provider.request(.zen) { _ in done() }
                         }
 
-                        expect(log).to(contain("Request:"))
-                        expect(log).to(contain("{ URL: https://api.github.com/zen }"))
+                        expect(log).to(contain("Request: https://api.github.com/zen"))
```

Comment:
The request's description wasn't correct here. I don't think it's due to my work but rather an update of AF 5 that wasn't fixed.